src/lib/djerba/helpers/expression_helper/helper.py

```diff
@@ -0,0 +1,122 @@
+"""Helper to write expression data, for use by SNV/indel and CNV plugins"""
+
+import csv
+import gzip
+import logging
+import os
+import djerba.util.constants as constants
+import djerba.core.constants as core_constants
+from djerba.helpers.provenance_helper.helper import main as fpr_helper
```

Comment:
- We aren't running the `provenance_helper`, but want to access one of its constants
- Making a cleaner way of doing this is TODO, but this works for now